delfin/drivers/dell_emc/vmax/rest.py

```diff
@@ -102,24 +110,43 @@ def request(self, target_uri, method, params=None, request_object=None):
 
         url, message, status_code, response = None, None, None, None
         if not self.session:
-            self.session = self._establish_rest_session()
+            self.session = self.establish_rest_session()
 
         try:
             url = ("%(self.base_uri)s%(target_uri)s" % {
                 'self.base_uri': self.base_uri,
                 'target_uri': target_uri})
-
-            if request_object:
-                response = self.session.request(
-                    method=method, url=url,
-                    data=json.dumps(request_object, sort_keys=True,
-                                    indent=4))
-            elif params:
-                response = self.session.request(
-                    method=method, url=url, params=params)
-            else:
-                response = self.session.request(
-                    method=method, url=url)
+            try:
+                if request_object:
+                    response = self.session.request(
+                        method=method, url=url,
+                        data=json.dumps(request_object, sort_keys=True,
+                                        indent=4))
+                elif params:
+                    response = self.session.request(
+                        method=method, url=url, params=params)
+                else:
+                    response = self.session.request(
+                        method=method, url=url)
+
+            except r_exc.SSLError:
```

Comment:
is it same as requests.exceptions.SSLError ? Can't we capture this   in driver.api.py for each driver interfaces and retry after relaod_certificate(). by this way we can make this opration to to the driver API layer instead if driver implemenation.